native/core/src/common/buffer.rs

```diff
@@ -202,7 +165,7 @@ impl CometBuffer {
     /// with 0. if `new_capacity` is less than the current capacity of this buffer, this is a no-op.
     #[inline(always)]
```

Comment:
I'm always a bit skeptical when I see an `#[inline(always)]` but this instance seems to pass the smell test. I know it's not your code, just noticing it in the diff.